src/test/kotlin/org/taymyr/lagom/elasticsearch/SampleDocument.kt

```diff
@@ -53,4 +54,21 @@ data class MessageMatchPhrase(val message: String) : MatchPhrase
 data class MessageMatchPhrasePrefix(val message: String) : MatchPhrasePrefix
 data class MessagePrefix(val message: String) : Prefix
 data class MessageWildcard(val message: String) : Wildcard
-data class MessageRegexp(val message: String) : Regexp
\ No newline at end of file
+data class MessageRegexp(val message: String) : Regexp
+
+data class SampleDocumentNestedObj(
```

Comment:
I think, that we no need to test ElasticSearch. We need to test the client.
Just create index and check, that created a mapping is correct.